smoke-test/build.gradle

```diff
@@ -135,6 +135,21 @@ task cypressDev(type: Exec, dependsOn: [installDev, ':metadata-ingestion:install
     environment 'DATAHUB_KAFKA_SCHEMA_REGISTRY_URL', 'http://localhost:8080/schema-registry/api/'
     environment 'KAFKA_BROKER_CONTAINER', 'datahub-kafka-broker-1'
 
+    workingDir = project.projectDir
+    commandLine 'bash', '-c',
+            "source ${venv_name}/bin/activate && set -x && " +
+                    "./cypress-dev.sh"
+}
+
+/**
+ * The following will install Cypress data in an already running stack.
+ */
+task cypressData(type: Exec, dependsOn: [installDev, ':metadata-ingestion:installDev']) {
+    environment 'RUN_QUICKSTART', 'false'
+    environment 'DATAHUB_KAFKA_SCHEMA_REGISTRY_URL', 'http://localhost:8080/schema-registry/api/'
+    environment 'KAFKA_BROKER_CONTAINER', 'datahub-kafka-broker-1'
+    environment 'RUN_UI', 'false'
```

Comment:
should this change be separated out?

Reply:
just a helper method to get more data for test easier

Reply:
Just a helper to get the cypress data loaded without running cypress